File: source/back-end/src/main/java/be/pxl/backend/service/ICityService.java
package be.pxl.backend.service;

import java.util.List;
import be.pxl.backend.entity.City;

public interface ICityService {

	public City find(String postal);
	
	public List<City> all();
	
	public void persist(City city);
	
	public void delete(String postal);
	
	public void update(String postal, City city);
	
}

File: Source/frontend/frontend.Service/ICityService.cs
using frontend.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace frontend.Service
{
    public interface ICityService
    {
        List<City> All();
        City Find(int id);
        void Add(City city);
        void Update(City city);
        void Delete(int id);
    }
}
